bin/kano-sync

```diff
@@ -138,7 +138,14 @@ def do_sync(skip_kdesk_refresh=False, dialog=True):
     if not skip_kdesk_refresh:
         refresh_kdesk()
 
+    save_profile_variable(
```

Comment:
Not quite sure why this was moved or if it won't set the variable now at every sync since this was previously wrapped in a `if not first_sync_done:`.